### PR TITLE
Gundam origins

### PR DESCRIPTION
**Pull request by chao-xian**
Blargh.

### CODE REVIEW

File: _posts/2017-04-06-gundam-origins-vol-1.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Confused Dan
+
+Dan echoed much of the same points as Jake, citing his loss of sense of scale in the opening scene. He didn't realise the Zaku were giant robots at first. He was also unsure who were the goodies and the baddies.
+
+He found some of the art absolutely gorgeous and looked absolutely amazing, but Dan wanted to understand what was going on.
+
+He supposed that it could be good as a dark military story and quite hard hitting war story, and in some cases it was, but then red ferret with his super cape turns up (I'm assuming he means Char) and Dan found that he couldn't stop laughing.
+
+Some not particularly spectacular to truly spectacular - it feels like it's of its time although it's like Star Wars it drawn as the future and cool but you can feel that it was done in the late 70s/80s
```
Comment (dandineen) [inline _posts/2017-04-06-gundam-origins-vol-1.md]: This paragraph doesn't make any sense. Sorry bud!

Comment (chao-xian) [inline _posts/2017-04-06-gundam-origins-vol-1.md]: Ya know, that's lifted from my notes where I was basically transcribing what people said. I stopped editing at one point 😂

Comment (dandineen) [inline _posts/2017-04-06-gundam-origins-vol-1.md]: Yeah, I sensed that :-P

Give me the heads up when it's less mangled and I'll get this up and live. I also need to do Tom's last write-up *and* mine too. Blimey!

**Comment by dandineen** [comment]
Do you wanna squash and then I'll merge it. Trying to be all professional like here ;-)

**Comment by dandineen** [comment]
But... but... process... :-P

**Comment by chao-xian** [comment]
@dandineen more than enough work as it is!

**Pull request merged**